src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchBox">
+      <form
+        action="./"
+        method="get"
+        class="bigSearchBar"
+        data-qa="big"
+      >
+        <input
+          type="search"
+          class="searchInput bigSearchInput"
```

Comment:
Add default style for searchInput class (base class for inputs) and change this style for smallSearchInput class (you don't need bigSearchInput class)